### PR TITLE
Added go fmt to test.sh

### PR DESCRIPTION
**Pull request by evanSpendlove**
One change: added `go fmt` to the test.sh script, so that it formats all .go files.

### CODE REVIEW

File: test.sh [hunk 1]
```diff
@@ -27,5 +27,7 @@ protoc -I=. --go_out=. github.com/googleinterns/step224-2020/config/proto/*.prot
 # Find finds all of the files ending in _test.go and returns their relative
 # paths. Sed removes the the file part of that relative path.
 # Then, we remove uniques.
-go test $(find ./github.com/googleinterns/step224-2020
- -type f -name '*_test.go' | sed -r 's|/[^/]+$||' -u)
+go test $(find ./github.com/googleinterns/step224-2020 -type f -name '*_test.go' | sed -r 's|/[^/]+$||' -u)
```
Comment (cianbgoog) [inline test.sh]: Alternative thing, that shouldn't be a blocker to merging this:

There must be a better way to deal with this :-P

**Review by cianbgoog — APPROVED** [review]

**Review by deniadelcid — APPROVED** [review]

File: test.sh (hunk 1 above)
Comment (evanSpendlove) [reply test.sh]: I have added a new version :-P

File: test.sh [hunk 2]
```diff
@@ -24,10 +24,7 @@ cd $HOME/go/src
...
 
 # Run all go tests
-# Find finds all of the files ending in _test.go and returns their relative
-# paths. Sed removes the the file part of that relative path.
-# Then, we remove uniques.
-go test $(find ./github.com/googleinterns/step224-2020 -type f -name '*_test.go' | sed -r 's|/[^/]+$||' -u)
+go test github.com/googleinterns/step224-2020/...
 
 # Run go fmt on all .go files to format them
 go fmt github.com/googleinterns/step224-2020/...
```
Comment (deniadelcid) [inline test.sh]: I would swap the order of these two: first run gofmt and then run tests.

**Review by deniadelcid — APPROVED** [review]

File: test.sh (hunk 2 above)
Comment (evanSpendlove) [reply test.sh]: That's a good point, thanks, I've reordered these.

**Pull request merged**